packages/repository/src/__tests__/fixtures/mixins/baseEntity.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity} from '../../../index';
```

Comment:
Please either remove `index` (import from `'../../..`), or change the import to reference the specific file implementing the imported artifacts.

Reply:
It should be `import {Entity} from '../../..`